ts/api/ProxyApi.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * Proxy client using apisauce
+ */
+
+import apisauce from "apisauce";
+
+import { apiUrlPrefix as proxyBaseURL } from "../config";
+import { IInstallation, IApiProfile, WithOnlyVersionRequired } from "./types";
+
+const create = () => {
+  const api = apisauce.create({
```

Comment:
consider switch this to an Autorest client when the API Proxy will publish its swagger specs.